File: lib/java/src/com/eliudarudo/cliapp/player/Choice.java
package com.eliudarudo.cliapp.player;

public class Choice {
    String letter;
    String statement;

    public Choice(String letter, String statement) {
       this.letter = letter;
       this.statement = statement;
    }

    public String fetchStatement() {
        return this.statement;
    }

    public String fetchLetter() {
        return this.letter;
    }
}

File: server/src/lib/go/src/github.com/eliudarudo/go-cli-app/player/choice.go
package player

// Choice Struct
type Choice struct {
	Letter    string
	Statement string
}

// FetchStatement function
func (choice *Choice) FetchStatement() string {
	return choice.Letter
}

// FetchLetter function
func (choice *Choice) FetchLetter() string {
	return choice.Statement
}
